### PR TITLE
Add an abstract

### PR DESCRIPTION
**Pull request by cjpatton**

### CODE REVIEW

**Comment by kunal-talwar** [comment]
Looks good to me. The only change I would suggest is to replace "no single input measurement significantly impacts the value of the aggregate result." to "no single input measurement significantly impacts the distribution of the
aggregate result."

**Comment by cjpatton** [comment]
Done.

File: draft-wang-ppm-differential-privacy.md [hunk 1]
```diff
@@ -245,13 +245,21 @@ informative:
...
+Distributed Aggregation Protocol (DAP) are designed to achieve. This document
+describes a variety of DP mechansisms applicable to DAP, and, for a variety of
+common use cases, lifts DAP to a protocol that also achieves DP.
 
 --- middle
 
 # Introduction
 
+[TO BE REMOVED BY RFC EDITOR: The source for this draft and and the reference
+implementations of mechansims and policies can be found at
```
Comment (junyechen1996) [inline draft-wang-ppm-differential-privacy.md]:
```suggestion
implementations of mechanisms and policies can be found at
```

File: draft-wang-ppm-differential-privacy.md [hunk 2]
```diff
@@ -245,13 +245,21 @@ informative:
...
+https://github.com/wangshan/draft-wang-ppm-dap-taskprov.]
```
Comment (junyechen1996) [inline draft-wang-ppm-differential-privacy.md]:
```suggestion
https://github.com/wangshan/draft-wang-ppm-differential-privacy.]
```

**Review by junyechen1996 — APPROVED** [review]
Some typos, otherwise LGTM.

**Comment by cjpatton** [comment]
Squashed.

**Pull request merged**